modules/002-deckhouse/hooks/pkg/apis/validate_deckhouse_cm.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apis
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	kwhhttp "github.com/slok/kubewebhook/v2/pkg/http"
+	"github.com/slok/kubewebhook/v2/pkg/model"
+	kwhmodel "github.com/slok/kubewebhook/v2/pkg/model"
+	kwhvalidating "github.com/slok/kubewebhook/v2/pkg/webhook/validating"
```

Comment:
I remember @RomanenkoDenys complaining about this library. Could you please contact him and ask which library he thinks fits best.

Reply:
Or persuade him using the `slok/kubewebhook` library 🙁

Reply:
Ok, but this library is used by us before. In some images, deckhouse-config and so more

Reply:
No problems here. It was another case. We have discussed it and `slok/kubewebhook` is not a blocker